src/index.css

```diff
@@ -1,52 +1,21 @@
-.nav-item {
-  background-color: #535bf2;
-}
-
+@import url('https://fonts.googleapis.com/css?family=Roboto');
 
-li{
-  list-style-type: none;
-  display: inline-block;
-  padding-right: 40px;
-  text-decoration: none;
+body:hover{
+  background-color: aliceblue;
 }
 
-li h1 {
-  padding-right: 1000px;
+/* form */
+.search{
+  background-color: none;
```

Comment:
background-color does not have `none` property. You can remove it.

Reply:
okay